week-2-practise/level-2/6-map.js

```diff
@@ -5,7 +5,10 @@
 
 var percentages = [1, 23, 92, 18];
 
-console.log(percentagesFormatted);
+function percentagesFormatted(index) {
+  return index + "%";
+}
+console.log(percentages.map(percentagesFormatted));
```

Comment:
Good. But `percentagesFormatted` doesn't describe the function very well. It formats a single percentage value as a string. so `formatAsPercentage` might be a better name.

An array storing formatted values might be more suited to a name like `percentagesFormatted`.